ios/Podfile

```diff
@@ -6,4 +6,5 @@ install! 'cocoapods', :deterministic_uuids => false
 
 target 'KaMPKitiOS' do
     pod 'shared', :path => '../shared/'
+    pod 'SwiftLint'
```

Comment:
Does this actually add code to the project? I would have expected a build-only dependency to be added in a different place, but that's just my Gradle intuition and I don't know what the norm is for Cocoapods